jest.config.base.js

```diff
@@ -10,4 +10,5 @@ module.exports = {
   moduleDirectories: ['node_modules'],
   automock: false,
   testTimeout: 10000,
+  testEnvironment: 'node',
```

Comment:
I think this change is breaking the Playwright tests, such that it doesn't have the right browser environment.

Reply:
I changed the way to add that to make it work with both unit and e2e.